hi-ml-multimodal/src/health_multimodal/image/inference_engine.py

```diff
@@ -78,7 +79,7 @@ def get_projected_global_embedding(self, image_path: Path) -> torch.Tensor:
                  where joint_feature_dim is the dimensionality of the joint latent space.
         """
         input_image, _ = self.load_and_transform_input_image(image_path, self.transform)
-        projected_img_emb = self.model.forward(input_image).projected_global_embedding
+        projected_img_emb = self.model.forward(ImageModelInput(image=input_image)).projected_global_embedding
```

Comment:
Could we break this up into multiple lines? Creating the image model input first then doing the forward pass, etc.

Reply:
I suggest keeping the current API so we don't break it: https://github.com/microsoft/hi-ml/pull/806#discussion_r1124704433

Reply:
[Resolving as it's now outdated]